howtos/how_to_handle_service_interruption_and_failures.py

```diff
@@ -1,7 +1,7 @@
 """module to handle service interruption and failures"""
 
 from solace.messaging.messaging_service import MessagingService, ServiceInterruptionListener, ServiceEvent
-from solace.messaging.publisher.direct_message_publisher import PublishFailureListener
+from solace.messaging.publisher.direct_message_publisher import PublishFailureListener, FailedPublishEvent
```

Comment:
We dont need to add this in the import statement since its used in the _annotation_ deceleration of the variable 
 `failed_publish_event: 'FailedPublishEvent'` That is one way to tell the compiler that `failed_publish_event` is of type `FailedPublishEvent`. You can further look into "Python annotations". The are completely optional, i.e. introduced in python 3+, however it is good practice to import them to enable code auto completion

Reply:
Make sense, good to know about this auto-resolution offered by Python. For the sake of correctness and to avoid warnings - I would prefer to leave it there, but to be consistent with other references removing it.